dwave/system/testing.py

```diff
@@ -61,35 +66,58 @@ class MockDWaveSampler(dimod.Sampler, dimod.Structured):
     def __init__(self, broken_nodes=None, broken_edges=None,
                  topology_type='chimera',topology_shape=None, **config):
         
-        if topology_type == 'pegasus':
+        if topology_type == 'zephyr':
+            if topology_shape == None:
```

Comment:
[Always compare with `None` using `is`](https://www.python.org/dev/peps/pep-0008/#programming-recommendations).
```suggestion
            if topology_shape is None:
```

Reply:
I see someone made the same omission below.. Please fix those instances while at it.

Reply:
Yeah - it was me last time as well, I've had bugs arise because of this sloppiness, thanks for the reminder.